Added ActivityIndicator implementation for iOS (#2829)
beeware/toga: changes/2829.feature.rst

> Added implementation of ActivityIndicator for iOS using the UIActivityIndicatorView

Reviewer: ```suggestion
The ActivityIndicator widget is now supported on iOS.
```
